submissions/evgenii-del/oop-exercise/index.js

```diff
@@ -0,0 +1,96 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+
+   Code repository: https://github.com/evgenii-del/a-tiny-JS-world
+   Web app: http://crooked-discovery.surge.sh/
+   */
+
+// ======== OBJECTS DEFINITIONS ========
+// Define your objects here
+
+class Inhabitant {
+    constructor(species, name, gender, saying) {
+        this.species = species;
+        this.name = name;
+        this.gender = gender;
+        this.saying = saying;
+    }
+
+    toString(object) {
+        return [this.species, this.name, this.gender, this.saying].join(';');
+    }
+}
+
+class Man extends Inhabitant {
+    constructor(species, name, gender, saying, hands, legs) {
```

Comment:
As we are in a `Man` class we already know the species (that we can send as an argument/string to the `super`) and can skip asking a developer to pass the species when calling `new Man`.
Same is applicable to other classes definitions.